pyproject.toml

```diff
@@ -43,7 +43,8 @@ certifi = "2025.1.31"
 unoserver = "3.3.2"
 openpyxl = "^3.1"
 
-waterbutler = { git = "https://github.com/CenterForOpenScience/waterbutler.git", branch = "feature/buff-worms" }
+#waterbutler = { git = "https://github.com/CenterForOpenScience/waterbutler.git", branch = feature/buff-worms"  }
+waterbutler = { git = "https://github.com/Vlad0n20/waterbutler.git", branch = "feature/ENG-8500" }
```

Comment:
In addition, let's don't change this for now since the WB PR will be merged before this one. @felliott